samples/bluetooth/hci_rpmsg/src/main.c

```diff
@@ -33,8 +33,13 @@
 LOG_MODULE_REGISTER(LOG_MODULE_NAME);
 
 /* Configuration defines */
+#if !DT_HAS_CHOSEN(zephyr_ipc_shm)
+#error "Sample requires definition of shared memory for rpmsg"
```

Comment:
Maybe use `BUILD_ASSERT_MSG` instead?

Reply:
From build-time perspective this gives the same result, IMHO.
Also, I would think a BUILD_ASSERT_MSG would be useful, if we wanted to check at build-time whether a value is not proper, but the check here is whether a node exists or not.

 @joerchan maybe I am missing sthg, so please, elaborate :)

Reply:
No strong reason, just thought BUILD_ASSERT_MSG would be more appropriate as a compile time check. But I see that #error is used in many places as well.

Reply:
Yes, that is acceptable, AFAIK